implementation/jwt-auth/src/main/java/io/smallrye/jwt/auth/principal/DefaultJWTCallerPrincipal.java

```diff
@@ -15,11 +15,7 @@
  */
 package io.smallrye.jwt.auth.principal;
 
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
done

Reply:
@nelsongraca Just a minor, you should have amended this change to the previous commit, so that this fix is associated with a single commit. Please squash these commits into a single commit